package.json

```diff
@@ -9,8 +9,8 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "test": "npm run test-spelling",
-    "test-spelling": "find . -iname \"*.*md\" -not -path \"./CHANGELOG*\" -not -path \"./node_modules/*\" | ./tools/test-spelling.sh",
+    "test": "npm run test:spelling",
+    "test:spelling": "cspell lint --no-summary --no-progress --gitignore --exclude '**/*.svg' '**/*'",
```

Comment:
I think narrowing down cspell to just docs directories might be a better choice.

Reply:
Sure at the moment, we are running cspell on the entire docs using the wildcard `**/*`
Instead of doing that, I would say let's run the spellcheck on only the directories containing docs. 
Namely, `config, pages, shared, templates, README.md, CONTRIBUTING.md`

Reply:
Let me know if I forgot something

Reply:
There is a directory called content?